Find a functionally equivalent code:android.widget.FastScroller.transitionToHidden:COMMENT
Method Modifier: private     
Comment:/**
 * Shows nothing.
 */

@@ -9,12 +9,13 @@
     mDecorAnimation = new AnimatorSet();
     mDecorAnimation.playTogether(fadeOut, slideOut);
     mDecorAnimation.start();
+    mShowingPreview = false;
 }

